packages/amplify-provider-awscloudformation/src/resource-package/resource-export.ts

```diff
@@ -431,7 +431,7 @@ export class ResourceExport extends ResourcePackager {
         // remove url parameters will set it in the construct
         // remove template URL the stack, CDK will update the URL
         if (deleteParameters) {
-          const { Parameters, TemplateURL, ...others } = template.Resources[resourceKey].Properties;
+          const { Parameters, ...others } = template.Resources[resourceKey].Properties;
```

Comment:
I'm not sure if it matters, but I think this changes the the behavior on line 437. By pulling out `TemplateURL` into it's own variable, it's being excluded from `resource.Properties`. Same with change on line 445.

Reply:
I'll descope it then for now.